### PR TITLE
[TACHYON-812] Add more test coverage for TieredBlockStore

### PR DESCRIPTION
**Pull request by yupeng9**
https://tachyon.atlassian.net/browse/TACHYON-812

Added test cases for TieredBlockStore.commitBlock

### CODE REVIEW

File: servers/src/test/java/tachyon/worker/block/TieredBlockStoreTests.java [hunk 1]
```diff
@@ -335,4 +335,32 @@ public void abortCommitedBlockTest() throws Exception {
     mBlockStore.commitBlock(USER_ID1, TEMP_BLOCK_ID);
     mBlockStore.abortBlock(USER_ID1, TEMP_BLOCK_ID);
   }
+
+  @Test
+  public void commitBlockTwice() throws Exception {
```
Comment (apc999) [inline servers/src/test/java/tachyon/worker/block/TieredBlockStoreTests.java]: `commitBlockTwice` -> `commitBlockTwiceTest`
This applies to other testcase name too

Comment (yupeng9) [inline servers/src/test/java/tachyon/worker/block/TieredBlockStoreTests.java]: bummer, my bad

Comment (yupeng9) [inline servers/src/test/java/tachyon/worker/block/TieredBlockStoreTests.java]: Done

**Comment by apc999** [comment]
+1 LGTM

File: servers/src/test/java/tachyon/worker/block/TieredBlockStoreTests.java [hunk 2]
```diff
@@ -335,4 +335,32 @@ public void abortCommitedBlockTest() throws Exception {
     mBlockStore.commitBlock(USER_ID1, TEMP_BLOCK_ID);
     mBlockStore.abortBlock(USER_ID1, TEMP_BLOCK_ID);
   }
+
+  @Test
+  public void commitBlockTwiceTest() throws Exception {
+    mThrown.expect(AlreadyExistsException.class);
+    mThrown.expectMessage(
+        "checkTempBlockOwnedByUser failed: blockId " + TEMP_BLOCK_ID + " is committed");
+    TieredBlockStoreTestUtils.createTempBlock(USER_ID1, TEMP_BLOCK_ID, BLOCK_SIZE, mTestDir1);
```
Comment (calvinjia) [inline servers/src/test/java/tachyon/worker/block/TieredBlockStoreTests.java]: In the other tests you usually put an empty line between the test setup and actual test operations?

Comment (yupeng9) [inline servers/src/test/java/tachyon/worker/block/TieredBlockStoreTests.java]: yep, done

**Comment by calvinjia** [comment]
Could you resolve the merge conflicts?

**Comment by calvinjia** [comment]
+1 LGTM

**Pull request merged**